lib/active_model/serializer.rb

```diff
@@ -266,5 +227,15 @@ def links
     protected
 
     attr_accessor :instance_options
+
+    private
+
+    def _serializer_instance_methods
```

Comment:
I think you're right.  I was working on the instance-level and got tunnel-vision